cli/chronos.py

```diff
@@ -75,3 +76,23 @@ def getCurrentImageVersion(self, roger_env, environment, application):
 
     def getTasks(self, roger_env, environment):
         print("Not yet implemented!")
+
+    def image_name(
```

Comment:
https://www.python.org/dev/peps/pep-0008/#function-names

Please see link and let me know your thoughts.